### ISSUE TITLE
Add daily average CPS

### ISSUE THREAD

**Issue by Slut** [running-coder/nanolooker#13]
Given that you provide confirmed transactions in the last 24 hours, could you also add the average CPS? It would be an interesting metric.

**Comment by tombertrand** [comment]
Right, I was considering saving the network's average / NanoLooker node's average  (daily averages) and have historical data, that would indeed be very interesting metrics over time. I may write a script and try to get back those values retrospectively from ~1 year ago but that will take some time
